### DISCUSSION TITLE
Where should you put images to export them to dist folder on build?

### DISCUSSION THREAD

**Discussion by ahku** [slidevjs/slidev#180] [Q&A] [answered]
Is there a way to import images so that they will be included in the dist build? 

Background: I'm building the presentations I make and upload them to an FTP folder, so there are a two manual steps I need to go through:

1. I build the presentation with --base=[target-path]
2. I copy the asset image to the assets folder

Since I reuse the same slidev setup, I also find myself "archiving" the slide by copying to an "archived" folder and renaming it to the date.

Is there a better workflow planned? Like being able to run slidev with a specific markdown file, or configuring multiple slides?

**Comment by antfu** [comment, marked as answer]
https://sli.dev/custom/directory-structure.html#public

> Like being able to run slidev with a specific markdown file,

`slidev foo.md` `slidev build foo.md`

> configuring multiple slides?

https://github.com/slidevjs/slidev/issues/53
